sdk/servicebus/azure-servicebus/azure/servicebus/management/_utils.py

```diff
@@ -44,3 +60,110 @@ def get_next_template(list_func, *args, **kwargs):
+            .find(constants.RULE_FILTER_TAG) \
+            .find(constants.RULE_FILTER_COR_PROPERTIES_TAG)
+        if correlation_filter_properties_ele:
+            deserialize_key_values(correlation_filter_properties_ele, rule_description.filter.properties)
+        sql_filter_parameters_ele = content\
+            .find(constants.RULE_DESCRIPTION_TAG) \
+            .find(constants.RULE_FILTER_TAG) \
+            .find(constants.RULE_PARAMETERS_TAG)
+        if sql_filter_parameters_ele:
+            deserialize_key_values(sql_filter_parameters_ele, rule_description.filter.parameters)
+        sql_action_parameters_ele = content\
+            .find(constants.RULE_DESCRIPTION_TAG) \
+            .find(constants.RULE_ACTION_TAG) \
+            .find(constants.RULE_PARAMETERS_TAG)
+        if sql_action_parameters_ele:
+            deserialize_key_values(sql_action_parameters_ele, rule_description.action.parameters)
+
+
+def serialize_key_values(root, kvs):
+    root.clear()
+    if kvs:
+        for key, value in kvs.items():
+            value_type, value_in_str = serialize_value_type(value)
+            key_value_ele = SubElement(root, QName(constants.SB_XML_NAMESPACE, constants.RULE_KEY_VALUE))
+            key_ele = SubElement(key_value_ele, QName(constants.SB_XML_NAMESPACE, constants.RULE_KEY))
+            key_ele.text = key
+            type_qname = QName(constants.XML_SCHEMA_INSTANCE_NAMESPACE, "type")
+            value_ele = SubElement(
+                key_value_ele, QName(constants.SB_XML_NAMESPACE, constants.RULE_VALUE),
+                {type_qname: constants.RULE_VALUE_TYPE_XML_PREFIX + ":" + value_type}
+            )
+            value_ele.text = value_in_str
+            value_ele.attrib["xmlns:"+constants.RULE_VALUE_TYPE_XML_PREFIX] = constants.XML_SCHEMA_NAMESPACE
+
+
+def serialize_rule_key_values(entry_ele, rule_descrpiton):
+    content = entry_ele.find(constants.CONTENT_TAG)
+    if content:
+        correlation_filter_parameters_ele = content\
+            .find(constants.RULE_DESCRIPTION_TAG) \
+            .find(constants.RULE_FILTER_TAG) \
+            .find(constants.RULE_FILTER_COR_PROPERTIES_TAG)
+        if correlation_filter_parameters_ele:
+            serialize_key_values(correlation_filter_parameters_ele, rule_descrpiton.filter.properties)
+        sql_filter_parameters_ele = content\
+            .find(constants.RULE_DESCRIPTION_TAG) \
+            .find(constants.RULE_FILTER_TAG) \
+            .find(constants.RULE_PARAMETERS_TAG)
+        if sql_filter_parameters_ele:
+            serialize_key_values(sql_filter_parameters_ele, rule_descrpiton.filter.parameters)
+        sql_action_parameters_ele = content\
+            .find(constants.RULE_DESCRIPTION_TAG) \
+            .find(constants.RULE_ACTION_TAG) \
+            .find(constants.RULE_PARAMETERS_TAG)
+        if sql_action_parameters_ele:
+            serialize_key_values(sql_action_parameters_ele, rule_descrpiton.action.parameters)
```

Comment:
descrption->description.  (would do as a github suggestion but don't think it can be multi-line?)

Reply:
Good catch! Updated.